elements/pl-checkbox/pl-checkbox.py

```diff
@@ -295,6 +300,8 @@ def render(element_html, data):
                 if answer_item['display_score_badge']:
                     answer_item['correct'] = (submitted_key in correct_keys)
                     answer_item['incorrect'] = (submitted_key not in correct_keys)
+                answer_item['display_feedback'] = submitted_answer['feedback']
+                answer_item['feedback'] = submitted_answer['feedback'] if submitted_answer['feedback'] else None
```

Comment:
You'll need to use `submitted_answer.get('feedback', None)` instead of `submitted_answer['feedback']` for backwards compatibility with existing question instances - this will fail with a `KeyError` otherwise.

Reply:
That should work, to be consistent with `pl-multiple-choice` though I'll modify this to use the `partial_scores` dictionary.